lib/pages/splash/splash_page.dart

```diff
@@ -1,40 +1,134 @@
+import 'dart:ui';
+
 import 'package:flutter/material.dart';
-import 'package:flutter/scheduler.dart';
 import 'package:flutter/widgets.dart';
+import 'package:flutter_svg/svg.dart';
 import 'package:get/get.dart';
+import 'package:survey/gen/assets.gen.dart';
 import 'package:survey/navigator/navigator.dart';
 import 'package:survey/pages/splash/splash_controller.dart';
 
-class SplashPage extends StatelessWidget {
-  final AppNavigator _appNavigator = Get.find();
-  final splashController = Get.put(SplashController());
+class SplashPage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _SplashPageState();
+}
+
+class _SplashPageState extends State<SplashPage> with TickerProviderStateMixin {
+  bool _isLogoAnimated = false;
+
+  AnimationController _logoOpacityController;
+  AnimationController _backgroundOpacityController;
```

Comment:
Please add `dispose` for these controllers 🙋

Reply:
thanks, I added to `dispose()` @manh-t :bowtie: